SU-SOE/stanford_soe_helper
# converted png social icons to svg and remove underline from dm menu

# READY FOR REVIEW

# Summary
- Converted .png files to .svg files for (icons were uploaded directly to the file directory - thanks, GG!)
- Removed underline from "Research & Ideas" DM menu title

# Needed By (Date)
- Monday/code push to prod

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Verify that Facebook, Twitter, and LinkedIn icons on Magazine Article nodes are loading .svg files
3. Verify that the "Research & Ideas" linked heading is not underlined

# Affected Projects or Products
- SOE
- stanford_soe_helper

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOE-2074

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
